commons_compress_7z/src/main/java/com/amaze/filemanager/filesystem/compressed/sevenz/SevenZFile.java

```diff
@@ -2117,17 +2119,27 @@ public String getDefaultName() {
     return lastSegment + "~";
   }
 
+  @Nullable
```

Comment:
Prefer not using `@Nullable` that is not from official Android SDK. Better pull it off at all.